app/index.jade

```diff
@@ -112,13 +112,54 @@ block content
           
         form.l-join-form(action='https://docs.google.com/forms/d/1w-8-OnbACv5M7c4KEHJtjXnDGwiPj_vnf7Lw9ereVow/formResponse', method='POST')
           label(for='your-name') name
-          input#your-name(type='text', required='', name='entry.372718999', placeholder='Jane Smith', autocomplete='name')
+            span.required *
+            input#your-name(type='text' name='entry.372718999', placeholder='Jane Smith', required)
           label(for='your-email') email
-          input#your-email(type='email', required='', name='entry.56111174', placeholder='name@address.com', autocomplete='email')
+            span.required *
+            input#your-email(type='email', name='entry.56111174', placeholder='name@address.com', required)
           label(for='your-organization') organization
-          input#your-organization(type='text', required='', name='entry.839901740', placeholder='Company Y')
+            input#your-organization(type='text', name='entry.839901740', placeholder='Company Y')
           label(for='your-role') role
-          input#your-role(type='text', required='', name='entry.790371713', placeholder='UX Designer')
+            input#your-role(type='text', name='entry.790371713', placeholder='UX Designer')
           label(for='your-reason') why you want to join
-          textarea#your-reason(name='entry.1562080588', placeholder="Please explain why you would like to join our team and why you want to help solve the refugee crisis...")
+            span.required *
+            textarea#your-reason(name='entry.1562080588', placeholder="Please explain why you would like to join our team and why you want to help solve the refugee crisis...", required)
           input(type='submit' value='submit' class="button")
+
+block page-script
```

Comment:
When running `gulp serve`, I get the following warning: `Warning: Unexpected block "page-script"  on line 129 of /Users/johnellison/Dropbox/John/Working/Sites/solveTheRefugeeCrisis.com/app/index.jade. This block is never used. This warning will be an error in v2.0.0`.

I am going to merge this and add the 'page-script' block to the default.jade layout file to get rid of the error message...